src/app/app.module.ts

```diff
@@ -45,10 +45,13 @@ import {environment} from '../environments/environment';
 import { MatIconModule, MatIconRegistry } from '@angular/material/icon';
 import {AdminModule} from './admin/admin.module';
 import {CookieService} from 'ngx-cookie-service';
+import { HelpComponent } from './help/help.component';
+import { MarkdownModule} from 'ngx-markdown';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    HelpComponent
```

Comment:
I would prefer to make this a separate module. That way it will be easier to include, replace or remove the feature in the future, if necessary. `AppModule` really is just tying things together without providing own functionality. In the future we might have separate AppModule implementations for different user organisations also.

Reply:
Ok, I created a new module and moved the component in it.